src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -85,7 +85,27 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
           disabled={!canJurorVote}
           onClick={() => onRequestCommit(VOTE_OPTION_REFUSE)}
         >
-          Refuse to vote
+          <span
+            css={`
+              margin-right: ${1 * GU}px;
+            `}
+          >
+            Refuse to vote
+          </span>
+          <div
+            onClick={event => {
+              event.stopPropagation()
+            }}
+          >
+            <Help hint="Why would I refuse to vote?">
```

Comment:
@bpierre I wonder if we can add the ability to activate `Help` components on hover for cases like this (and disable their onclicks).

It does not seem like a great pattern to have to click to open this Help tooltip if it's inside a button.

Reply:
Agree on that i think that it might not make sense to click on a button tooltip

Reply:
I’m wondering if it makes sense to have this opener inside of the button? Even if we do it on hover, it wouldn’t work for touch-only screens where we would still have to click on it. /cc @dizzypaty